src/state/slices/opportunitiesSlice/resolvers/foxFarming/index.ts

```diff
@@ -115,9 +118,9 @@ export const foxFarmingLpMetadataResolver = async ({
       [opportunityId]: {
         apy,
         assetId: opportunityId,
-        provider: DefiProvider.FoxFarming,
+        provider: DefiProvider.FoxFarming as const,
```

Comment:
I'm not sure we need these? Typescript is happy without it.

Reply:
3a39e20ea138319032420d79d970c7900e3c167f